### PR TITLE
2 New Projects added

### PR DESCRIPTION
**Pull request by Aryan-karawale**

### CODE REVIEW

**Comment by SuperChamp234** [comment]
@Aryan-karawale please make suggested changes ASAP

File: _projects/ctRL.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: ctTL
```
[the remark this answers is not in the document]
Comment (Aryan-karawale) [reply _projects/ctRL.md]: Did That Please check

File: _projects/Talking_Lamdas.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Talking_Lamdas
```
[the remark this answers is not in the document]
Comment (Aryan-karawale) [reply _projects/Talking_Lamdas.md]: We are going with ROSfunc for now

[the remark this answers is not in the document]
Comment (SuperChamp234) [reply _projects/Talking_Lamdas.md]: Great

File: _projects/ctRL.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SuperChamp234) [reply _projects/ctRL.md]: Great

**Review by SuperChamp234 — APPROVED** [review]

**Comment by SuperChamp234** [comment]
LGTM!

**Pull request merged**